Documentation/hetzner-cicd.md

```diff
@@ -42,6 +42,10 @@ in Hetzner hosts, both bare-metal and VMs.
   ssh -J root@49.13.211.148 root@192.168.84.3
   ```
 
+### Hetzner VM for Zephyr testing
+* [Internal documentation](https://github.com/NorthernTechHQ/sre-iac/tree/main/hetzner-zephyr-qa)
```

Comment:
This repository does not exist (?)

Reply:
Now you should be able to see the repo, not yet the specific file, which is related to this PR: https://github.com/NorthernTechHQ/sre-iac/pull/14